src/components/Button/Button.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { h, FunctionComponent } from 'preact';
+
+interface ButtonProps {
+  label?: string;
+  size?: 'md' | 'sm' | 'full';
+  onClick?: () => void;
+  classname?: string;
+  color: string;
+  buttonPosition?: 'left' | 'right' | 'center';
+  type?: 'button' | 'submit' | 'reset';
+}
+
+export const Button: FunctionComponent<ButtonProps> = ({
+  label,
+  size = 'md',
+  onClick,
+  classname,
+  type,
+  color,
+  buttonPosition = 'left',
+}) => {
+  const onClickHandler = () => {
+    if (onClick) {
+      onClick();
+    }
+  };
+  let widthClasses = 'w-auto';
+
+  if (size === 'md') {
+    widthClasses = 'w-32';
+  } else if (size === 'sm') {
+    widthClasses = 'w-fit';
+  } else if (size === 'full') {
+    widthClasses = 'w-full';
+  }
+
+  const buttonPositionClasses =
+    buttonPosition === 'left'
+      ? 'float-left'
+      : buttonPosition === 'right'
+      ? 'float-right'
```

Comment:
Using `float` without clearfix would lead to elements flowing around the floated element, do we really need the `buttonPosition` prop?
I think we can do without the `buttonPosition` prop, we can just nest `Button` in a flex element whenever it's needed on the right and set `justify-content` to `flex-end.` , What do you think?